data/data-pipeline/tox.ini

```diff
@@ -15,7 +15,8 @@ commands = black data_pipeline
 [testenv:checkdeps]
 # checks the dependencies for security vulnerabilities and open source licenses
 allowlist_externals = bash
-commands = safety check --ignore 44715  # known issue: https://github.com/pyupio/safety/issues/364
+commands = pip install -U wheel
```

Comment:
Does this mean that we removed it from GHA and placed wheel into tox instead?

Reply:
Tox sets up its own separate virtualenv from the main one, so I neded to replicate upgrading wheel for tox runs the way I did in the general action.